projects.md

```diff
@@ -0,0 +1,10 @@
+# Projects
+
+Existing projects in Go that can be used as examples.
+
+## HelloFresh
+
+- https://github.com/hellofresh/remote-config-service
```

Comment:
Either this repo becomes private or we have to find a way to have a service directory that you can look at.